R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -141,3 +141,24 @@ test_that("kmeans", {
   cluster <- summary.model$cluster
   expect_equal(sort(collect(distinct(select(cluster, "prediction")))$prediction), c(0, 1))
 })
+
+test_that("SparkR::survreg vs survival::survreg", {
+  library(survival)
```

Comment:
actually, `suggests` means it will not be installed automatically, which I think is right - I don't think we should require `survival` or `e1071` to use SparkR
http://r-pkgs.had.co.nz/description.html